site/class-1-load/leonvanzijl/leonvanzijl/index.csv

```diff
@@ -0,0 +1,4 @@
+day,dinner
```

Comment:
You’re uploading your files in `site/class-1-load/leonvanzijl/leonvanzijl/` but they should be added to `site/class-1-load/leonvanzijl/`.

Reply:
...and use absolute dates in your CSV!

Reply:
uploaded the files directly in site/class-1-load/leonvanzijl/ and changed the dates in the CSV / Json

Reply:
That looks great, but you still have the extra directory. Maybe remove this branch and try again?